dashboard/app/models/unit.rb

```diff
@@ -721,6 +721,13 @@ def can_view_version?(user, locale: nil)
     user.assigned_script?(self)
   end
 
+  def next_unit(user)
```

Comment:
Would you mind adding a comment for the method please? Just in case its function is needed elsewhere too!